src/components/tracked-controls.js

```diff
@@ -156,7 +164,9 @@ module.exports.Component = registerComponent('tracked-controls', {
       object3D.position.fromArray(pose.position);
     } else {
       // Controller not 6DOF, apply arm model.
-      if (this.data.armModel) { this.applyArmModel(object3D.position); }
+      if (this.data.armModel) {
+        this.applyArmModel(object3D.position);
```

Comment:
Can we keep this as a one liner?

Reply:
Sure, what is the rationale for 1 line versus line breaking? there seems to be no formatter or consistency across the code base with regards to this. The only consistency I did find was that most complex statements (like this one) are broken across lines and return clauses within the statement block tend to be on the same line.

Reply:
You're right that the rule is a bit loose. If the line is short we keep it in one line. If you have a very long statement we break it in multiple lines. We don't have a specific number of characters. That's the best answer I can give 😄